### ISSUE TITLE
Should 'any' work?

### ISSUE THREAD

**Issue by k1sul1** [CalderaWP/REST-API-Multiple-Post-Types#1]
As the title says. Sometimes you want to query with IDs directly, and don't care about or don't even know the post type.

I tried and it didn't work.

This can be done pretty easily on the frontend assuming you have fetched the post types from the API, but if this would be a nice to have feature. WP_Query has it :slightly_smiling_face:

**Comment by Shelob9** [comment]
No, it will not. Any is going to fail this conditional https://github.com/CalderaWP/REST-API-Multiple-Post-Types/blob/cd343cdddd17f06faa74ffa14524c85ac6f239c8/rest-multiple-post-types.php#L35

**Comment by k1sul1** [comment]
Do you think adding it would be a good feature? Shouldn't be too hard to implement and would make it cleaner to use in the frontend.

Right now we have `WP.getFromPostTypes(Object.keys(WP.postTypes), { include: [1, 2, 3] })` which works, but _eh_.

**Comment by Shelob9** [comment]
If it only worked for post types that had show_in_rest set to true, yes,
I'd accept a PR.
